src/ToDoList.js

```diff
@@ -18,177 +16,142 @@ import {
   IconButton,
 } from "@mui/material";
 
-const containerStyle = {
-  display: "flex",
-  justifyContent: "center",
-  alignItems: "center",
-  width: "500px",
-  height: "600px",
-  backgroundColor: "lightGrey",
-  borderRadius: "30px",
-  margin: "0 auto",
-  marginTop: "30px",
-};
-
-const titleStyle = {
-  color: "black",
-  variant: "h1",
-  width: "300px",
-  height: "50px",
-  //borderRadius: "20px",
-  //backgroundColor: "lightBlue",
-  margin: "0 auto",
-  fontWeight: "bold",
-  //justifyContent: "center",
-  //alignItems: "center",
-};
-
-const inputContainerStyle = {
-  display: "flex",
-  justifyContent: "center",
-  alignItems: "center",
-  //height: "40px",
-};
-
-/*const taskItemStyle = {
-  width: "200px",
-  display: "flex",
-  display: "flex",
-  justifyContent: "center",
-  alignItems: "center",
-};*/
-
-const completedTaskStyle = {
-  textDecoration: "line-through",
-  textDecorationStyle: "solid",
-  textDecorationThickness: "2px",
-  color: "black",
-};
-
-/*const deleteButtonStyle = {
-  fontSize: "10px",
-  padding: "3px 3px",
-};*/
+const containerStyle = css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 30px;
+  h1 {
+    color: #29b6f6;
+    font-size: 50px;
+    font-weight: bold;
+    margin-top: 30px;
+  }
+`;
+
+const completedTaskStyle = css`
+  text-decoration: line-through solid 2px;
+  color: black;
+`;
+
+const listContainerStyle = css`
+  width: 100%;
+  max-height: 450px;
+  overflow-x: hidden;
+  overflow-y: auto;
+`;
 
 const ToDoList = () => {
-  useEffect(() => {
-    const storedTasks = localStorage.getItem("tasks");
-    if (storedTasks) {
-      setTasks(JSON.parse(storedTasks));
-    }
-  }, []);
-
-  const [tasks, setTasks] = useState(() => {
-    const storedTasks = localStorage.getItem("tasks");
+  const [newTaskValue, setNewTaskValue] = useState(() => {
+    const storedTasks = localStorage.getItem("newTaskValue");
     return storedTasks ? JSON.parse(storedTasks) : [];
   });
 
-  const [taskInfo, setTaskInfo] = useState("");
-  const [filter, setFilter] = useState("all");
-
-  useEffect(() => {
-    localStorage.setItem("tasks", JSON.stringify(tasks));
-  }, [tasks]);
+  const [inputValue, setInputValue] = useState("");
+  const [taskState, associateTaskState] = useState("all");
```

Comment:
the convention for the `useState` array variables is that they should have the same name,
just that the setter function should have a `set` prefix that is followed by a capital letter.

for example:

`const [count, setCount] = useState(0);`

so, a better name for this use case would be:
`const [taskStateFilter, setTaskStateFilter] = useState(FilterByTaskStateOption.ALL);`

see a comment regarding `FilterByTaskStateOption` below.